docs/installation/autoware/docker-installation.md

```diff
@@ -4,150 +4,23 @@
 
     Since this page explains Docker-specific information, it is recommended to see [Source installation](./source-installation.md) as well if you need detailed information.
 
-## Prerequisites
+Here are two ways to install Autoware by docker:
 
-- [Git](https://git-scm.com/)
+- The first way is to start Autoware with `prebuilt image`, this is a quick start, this way you can only run Autoware simulator and not develop Autoware, it is only suitable for beginners
 
-- For NVIDIA Jetson devices, install [JetPack](https://docs.nvidia.com/jetson/jetpack/install-jetpack/index.html#how-to-install-jetpack) >= 5.0
+- The second way is to start Autoware with `devel image`, which supports developing and running Autoware using docker
 
-## How to set up a development environment
+## Docker installation for quick start
 
-1. Clone `autowarefoundation/autoware` and move to the directory.
+[docker installation for quick start](./docker-installation-prebuilt.md)
 
-   ```bash
-   git clone https://github.com/autowarefoundation/autoware.git
-   cd autoware
-   ```
+![type:video](https://drive.google.com/file/d/1P0hCJ8jQHRMQ1xGINDjqn4MQpiMOMVGd/view?usp=share_link)
```

Comment:
```suggestion
[Reference video tutorials](https://drive.google.com/file/d/1P0hCJ8jQHRMQ1xGINDjqn4MQpiMOMVGd/view?usp=share_link)
![type:video](https://www.youtube.com/embed/LXb3EKWsInQ)
```
Hmm the video tag didn't work on the google drive link.

![Screenshot from 2023-01-07 01-53-40](https://user-images.githubusercontent.com/10751153/211113288-6b6f4e73-8516-4d21-8b04-6a17517ddb64.png)

I want to test if it works at all with their example in https://pypi.org/project/mkdocs-video/